File: src/taburet/transactions/_design/transactions/views/balance/map.erl
fun({Doc}) ->
	case proplists:get_value(<<"doc_type">>, Doc) of
		<<"Transaction">> ->
			Amount = proplists:get_value(<<"amount">>, Doc),
			
			lists:foreach(
				fun(AccId)-> Emit(AccId, {[{debet, 0.0}, {kredit, Amount}]}) end,
				proplists:get_value(<<"from_acc">>, Doc)
			),
			
			lists:foreach(
				fun(AccId)-> Emit(AccId, {[{debet, Amount}, {kredit, 0.0}]}) end,
				proplists:get_value(<<"to_acc">>, Doc)
			);
		_ -> ok
	end
end.

File: taburet/transactions/_design/old_transactions/views/balance/map.js
function(doc) {
	if (doc.doc_type && doc.doc_type == 'Transaction') {
		doc.from_acc.forEach(function(e) {
			emit(e, {kredit:doc.amount, debet:0})
		})
		doc.to_acc.forEach(function(e) {
			emit(e, {kredit:0, debet:doc.amount})
		})
	}
}